### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by undefobj**

### CODE REVIEW

File: android/authentication.md [hunk 1]
```diff
@@ -631,16 +631,15 @@ In a terminal window, navigate to the root of your app files and add the auth ca
...
 ❯ No, I will set up my own configuration.
 ❯ User Sign-Up, Sign-In, connected with AWS IAM controls
 ```
 
 Choose **YES** to `? Allow unauthenticated logins?` and **YES** to `? Do you want to enable 3rd party authentication providers in your identity pool?`.
 
-Choose **Google** and then provide your Google **Client ID**.
-
-When configuration for Facebook sign-in is complete, the CLI displays a message confirming that you have configured local CLI metadata for this category. Run the following to update your changes in the cloud:
+Choose **Google** and then provide your Google **Client IDs** as appropriate. The CLI will as you for both the **Web Client ID** and **Android Client ID** at the appropriate time.
```
Comment (mlabieniec) [inline android/authentication.md]: Should this be "ask" you for ?

File: ios/authentication.md [hunk 2]
```diff
@@ -572,16 +572,16 @@ In a terminal window, navigate to the root of your app files and add the auth ca
...
-$ amplify add auth
+$ amplify add auth              ##"amplify update auth" if already configured
 ❯ No, I will set up my own configuration.
 ❯ User Sign-Up, Sign-In, connected with AWS IAM controls
 ```
 
 Choose **YES** to `? Allow unauthenticated logins?` and **YES** to `? Do you want to enable 3rd party authentication providers in your identity pool?`.
 
-Choose **Google** and then provide your Google **Client ID**.
+Choose **Google** and then provide your Google **Client IDs** as appropriate. The CLI will as you for both the **Web Client ID** and **iOS Client ID** at the appropriate time.
```
Comment (mlabieniec) [inline ios/authentication.md]: Same as above "ask"

Comment (undefobj) [reply ios/authentication.md]: fixed

File: android/authentication.md (hunk 1 above)
Comment (undefobj) [reply android/authentication.md]: fixed

**Pull request merged**